### DISCUSSION TITLE
No entiendo nada modulo 10

### DISCUSSION THREAD

**Discussion by ArmandoRiveraT** [LaunchX-InnovaccionVirtual/CursoIntroPython#256] [Q&A]
Por mas que ejecuto el codigo no me sale nada, alguien tendra link externo donde expliquen lo de esta clase? 🤯
solo me sale: 
PS C:\Users\pixel\Downloads\CursoIntroPython-main> & C:/Users/pixel/AppData/Local/Programs/Python/Python310/python.exe "c:/Users/pixel/Desktop/Nueva carpeta/config.py"
PS C:\Users\pixel\Downloads\CursoIntroPython-main> & C:/Users/pixel/AppData/Local/Programs/Python/Python310/python.exe "c:/Users/pixel/Desktop/Nueva carpeta/config.py"
PS C:\Users\pixel\Downloads\CursoIntroPython-main>

**Comment by ArmandoRiveraT** [comment]
<img width="960" alt="Captura de pantalla 2022-02-14 201750" src="https://user-images.githubusercontent.com/99206229/153979983-be230a60-43ae-4f6e-af0f-f7a5c98368cb.png">

**Reply by crizrom** [reply]
Me pasa lo mismo, el único que me funciona es PermissionError

**Reply by AbrahamDiaz1** [reply]
Necesitas poner al final del script config.py la parte 
<pre>if __name__ == '__main__':
    main()</pre>
porque es la parte que ejecuta la función main() cuando lo corres desde python ya que el script recibe el nombre `__name__` de `__main__` y entra en la condicional para ejecutar la función `main()`

De lo contrario solo crea la función pero nunca se ejecuta y de preferencia solo tener una función main que vas actualizando conforme el ejercicio, yo lo que hago es que todo lo meto a un notebook.

**Comment by EinarDvls** [comment]
hola compañero mira https://github.com/LaunchX-InnovaccionVirtual/CursoIntroPython/discussions/223 aquí un compañero nos explico como iba la cosa, espero pueda ayudarte.
